packages/picasso-rich-text-editor/src/RichTextEditor/RichTextEditor.tsx

```diff
@@ -31,7 +31,7 @@ export interface Props extends BaseProps {
   /** unique identifier */
   id: string
   /**
-   * @deprecated Use the `status` prop instead to both support success and error states
+   * @deprecated [FX-4715] Use the `status` prop instead to both support success and error states
```

Comment:
NIT, not related to changes in current pull request

```suggestion
   * @deprecated [FX-4715] Use the `status` prop instead to support success and error states
```